### PR TITLE
DDF-1897: Updated metacard versioning so that ingest is not a separat…

### PR DESCRIPTION
**Pull request by rzwiefel**
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@kcwire 
@pklinef

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

…e transaction

### CODE REVIEW

File: catalog/plugin/catalog-plugin-versioning/src/main/resources/OSGI-INF/blueprint/blueprint.xml [hunk 1]
```diff
@@ -24,6 +24,7 @@
     </bean>
 
     <service ref="historian" interface="ddf.catalog.plugin.PostIngestPlugin" />
+    <service ref="historian" interface="ddf.catalog.plugin.PreIngestPlugin" />
```
Comment (kcwire) [inline catalog/plugin/catalog-plugin-versioning/src/main/resources/OSGI-INF/blueprint/blueprint.xml]: You should just list both interfaces under the same service tag.

File: catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java [hunk 2]
```diff
@@ -46,7 +53,8 @@ public HistorianPlugin(CatalogFramework catalogFramework) {
 
     @Override
     public CreateResponse process(CreateResponse input) throws PluginExecutionException {
-        getVersionedMetacards(input.getCreatedMetacards(), HistoryMetacardImpl.Action.CREATED);
+//        getVersionedMetacards(input.getCreatedMetacards(), HistoryMetacardImpl.Action.CREATED);
```
Comment (kcwire) [inline catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java]: Remove commented code

File: catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java [hunk 3]
```diff
@@ -99,4 +107,44 @@ private void store(List<Metacard> metacards) throws PluginExecutionException {
...
+                .collect(Collectors.toList());
+        updatedMetacardList.addAll(input.getMetacards());
+        return new CreateRequestImpl(updatedMetacardList,
+                input.getProperties());
+    }
+
+    @Override
+    public UpdateRequest process(UpdateRequest input)
+            throws PluginExecutionException, StopProcessingException {
+        /*List<Metacard> updatedMetacardList = input.getUpdates()
```
Comment (kcwire) [inline catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java]: Remove

**Comment by pklinef** [comment]
:+1: after you fix Keith's comments

File: catalog/plugin/catalog-plugin-versioning/src/main/resources/OSGI-INF/blueprint/blueprint.xml (hunk 1 above)
Comment (rzwiefel) [inline catalog/plugin/catalog-plugin-versioning/src/main/resources/OSGI-INF/blueprint/blueprint.xml]: done

File: catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java (hunk 2 above)
Comment (rzwiefel) [inline catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java]: done

File: catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java (hunk 3 above)
Comment (rzwiefel) [inline catalog/plugin/catalog-plugin-versioning/src/main/java/ddf/catalog/metacard/versioning/HistorianPlugin.java]: done

**Comment by jlcsmith** [comment]
Full clean build passed.

**Comment by kcwire** [comment]
👍

**Pull request merged**